Form-Controls/index.html

```diff
@@ -13,6 +13,37 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+        <div>
+          <label for="name">Name</label>
+          <input type="name" name="name" id="name" minlength="2" required />
+        </div>
+
+        <div>
+          <label for="email">E-mail address</label>
+          <input type="email" name="email" id="email" />
+        </div>
+
+        <div>
+          <label for="colour">T-shirt colour</label>
+          <select name="colour" id="colour">
+            <option label="Red" value="Red"></option>
+            <option label="Green" value="Green"></option>
+            <option label="Blue" value="Blue"></option>
+          </select>
+        </div>
+
+        <div>
+          <label for="size">T-shirt size</label>
+          <select name="size" id="size">
+            <option label="XS" value="XS"></option>
+            <option label="S" value="S"></option>
+            <option label="M" value="M"></option>
+            <option label="L" value="L"></option>
+            <option label="XL" value="XL"></option>
+            <option label="XXL" value="XXL"></option>
+          </select>
+        </div>
+        <button type="submit">Submit</button>
```

Comment:
This is a good choice for the button - you can see that when you submit the form, the browser is validating the form for you.